.github/workflows/dotnet/pr_checks.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and Coverage
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main", "master", "stage", "dev", "release/*"]
+  pull_request:
+    branches: [ "main", "master", "stage", "dev", "release/*" ]
+    
+env:
+  DOTNET_VERSION: 7.0.x
+  COVERAGE_THRESHOLD: ${{vars.COVERAGE_THRESHOLD}}
+      
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
There is a newer checkout version: https://github.com/actions/checkout

Reply:
Done